pkg/plugins/message/marshaler_json.go

```diff
@@ -2,26 +2,56 @@ package message
 
 import (
 	"encoding/json"
+
+	cloudevents "github.com/cloudevents/sdk-go"
 )
 
+// Marshaler ...
+type Marshaler interface {
+	Marshal(interface{}) ([]byte, error)
+	Unmarshal([]byte, interface{}) error
```

Comment:
@andrewlouis93 changed the cloud events to internal representation.

Reply:
awesome 👍 ty!